src/vs/workbench/contrib/chat/browser/chatContentParts/toolInvocationParts/chatTerminalToolProgressPart.ts

```diff
@@ -130,6 +135,11 @@ class TerminalCommandDecoration extends Disposable {
 			}
 		}
 
+		if (!this._hover.value && this._currentHoverText) {
+			this._register(this._hoverService.setupDelayedHover(decoration, {
+				content: this._currentHoverText,
+			}));
+		}
```

Comment:
You should set this up when the element is ready and use a callback for the options if content will change:

```ts
(() => {
  content: this._currentHoverText
})
```

Otherwise content will be fixed to whatever it was when setup was called.